src/OpenTelemetry.Contrib.Instrumentation.AWS/README.md

```diff
@@ -0,0 +1,31 @@
+# AWS SDK client instrumentation for OpenTelemetry
```

Comment:
if we are setting precedence, this file may include information on how to get support for this package and who are maintainers for this package

Reply:
+1. I can do it for this project and Extensions.AWSXRay together in a PR